xAPI.md

```diff
@@ -675,6 +675,18 @@ LRS SHOULD* reject Statements containing such additional properties.
 
 <a name="statement-immutablity-and-exceptions" />
 
+Statements are evidence of a tracked learning experience.  Because this information is considered to be evidence, 
+the information represented in the Statement has already happened.  Thus, the natural language used in "display" or 
+in the human-readable portion of the verb will usually use the past tense.
+
+Because Statements are evidence, they are expected to be permanent.  The only way to undo a Statement within this 
+specification is to <a name="voided">"void" it</a>it.  Voiding does not destroy a Statement, rather indicates the 
+evidence in the Statement is to be disregarded.
+
+Just as evidence is to be shared among all authorized parties, it is expected that LRSs can perform various 
+operations to move Statements around as needed.  Details of those operations is beyond the scope of this 
+specification.  
```

Comment:
Can't seem to reply to the points above.  Here goes:  Does anything think that recommending the human-readable portion of the IRI (if it is a verb) being past tense is a bad thing (It isn't even a SHOULD as much as it is a recommendation).  Totally agree the document will need a comb for "past tense" I'll open an issue.  Can you elaborate on the purpose of GET Statements design and what this paragraph does to hurt that?  Thanks!!